docs/NPC_Swap_Tutorial.md

```diff
@@ -0,0 +1,224 @@
+---
+sidebar_position: 1
+description: An introductory modding guide that will show you how to swap 47's Signature Suit with an NPC.
+---
+
+# Recommended reading 
+* [Glacier 2 Modding Basics](https://wiki.notex.app/glacier2/modding_basics)
+* [Chunk Data](https://wiki.notex.app/glacier2/chunkdata)
+
+# Requirements
+* [RPKG Tool](https://notex.app/rpkg/)
+* [Simple Mod Framework](https://www.nexusmods.com/hitman3/mods/200)
+* A program to read and edit JSON files (VSCode is recommended, but any basic code editor will suffice in this scenario as we're only changing a single value at the beginning of the document. Otherwise, QuickEntity Editor is the superior choice when editing QN JSON files)
+* A program to extract and archive ZIP files (such as 7-Zip)
+
+# Background
+Replacing an outfit requires you to identify an outfit to be replaced as well as an outfit to be replaced by. For the purpose of this tutorial, we will attempt to replace 47’s Signature Suit with that of Silvio Caruso, who is already present in the game's files. 
+Creating a mod requires you to create your own `chunk` file, which you then ‘inject’ into the game's files. This can be done in multiple ways. This tutorial uses [Simple Mod Framework]( https://www.nexusmods.com/hitman3/mods/200) to automate this part, as the modding process is already convoluted enough without this tool.

```

Comment:
May be beneficial to explain how the patch system works. Other document maybe?

Reply:
Yeah, agreed. I was thinking of making this the first tutorial in a series, as I didn't want new users to be intimidated by too massive a wall of text haha. I'd also like to go over editing entities with QNE, modifying textures/meshes, etc.